src/seq/movement.rs

```diff
@@ -72,13 +73,14 @@ impl SeqMove {
     }
 
     fn get_dir(player: &Vector3<f32>, target: &Vector3<f32>, climb: bool) -> [f32; 2] {
-        let mut diff = *target - *player;
-        diff.normalize();
-        if climb {
+        let diff = *target - *player;
+        let mut diff2d = if climb {
             [diff.get_x(), diff.get_y()]
         } else {
             [diff.get_x(), diff.get_z()]
-        }
+        };
+        vec2::norm_mut(&mut diff2d);
+        diff2d
```

Comment:
I have no idea what any of this means

SHIP IT!

Reply:
Short of it is:

1. Calculate the 3d vector between the target point and the player's current position.
2. Create a 2d vector, discarding one of the axis (use y or z for the second depending on if we are climbing on a wall)
3. Normalize the result so the length of the 2d vector is 1 (which the joystick crate expects)